utils/scale-ci-diagnosis/ocp_diagnosis.sh

```diff
@@ -92,12 +92,15 @@ function capture_full_db() {
 
 
 function must_gather() {
-	oc adm must-gather -- bash -c 'gather && gather_network_logs && gather_network_ovn_trace'
-	mv must-gather* $OUTPUT_DIR/must-gather-$ts 
-	XZ_OPT=--threads=0 tar cJf $OUTPUT_DIR/must-gather-$ts.tar.xz $OUTPUT_DIR/must-gather-$ts
-	if [[ $? -eq 0 ]]; then
-		rm -rf $OUTPUT_DIR/must-gather-$ts
-	fi
+        oc adm must-gather -- bash -c 'gather && gather_network_logs && gather_network_ovn_trace'
+        if [[ $? -eq 0 ]]; then
+                mv must-gather* $OUTPUT_DIR/must-gather-$ts 
+                XZ_OPT=--threads=0 tar cJf $OUTPUT_DIR/must-gather-$ts.tar.xz $OUTPUT_DIR/must-gather-$ts
+                rm -rf $OUTPUT_DIR/must-gather-$ts
+        else
+                exit 0
```

Comment:
Can we please add a log here as well..that must-gather failed.... Moving to next step

Reply:
right , makes sense